api-tests/apply_route_test.go

```diff
@@ -343,6 +343,20 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 	}
 }
 
+func TestPmmServerUnimplemented(t *testing.T) {
```

Comment:
TestPmmServerUnimplemented -> TestPmmServerApplyUnimplemented?

It does not make sense to implement that endpoint imho, if we request it anyway, unimplemented grpc code is returned.